sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -520,6 +520,12 @@ def _add_argparse_args(cls, parser):
         type=str,
         help='GCE minimum CPU platform. Default is determined by GCP.'
     )
+    parser.add_argument(
```

Comment:
In light of the discussion here on the dev@ list related to runner options (https://lists.apache.org/thread.html/78fe33dc41b04886f5355d66d50359265bfa2985580bb70f79c53545@%3Cdev.beam.apache.org%3E). Would it be better to expose this as a runner option?

@robertwb